### ISSUE TITLE
when are features included in branch "0.13-LTS" applied to release version?

### ISSUE THREAD

**Issue by heechanlee** [bentoml/BentoML#2136]
First of all, Thanks for your efforts making this great application!

I have a little problem while using it.
I want to use custom prometheus metric using prometheus client described as below
![image](https://user-images.githubusercontent.com/65716726/146705215-a0fc5467-b433-43d1-862b-d27adbe8f0f1.png)

I can't use this feature because it is not yet applied to 0.13.1

when can I use this feature? or how can I use this feature now?

**Comment by aarnphm** [comment]
Hi @heechanlee, I believe that custom metrics should be supported in 0.13.1.

**Comment by heechanlee** [comment]
@aarnphm 
Thank you for reply!
Please be generous for my misunderstanding if any..

I checked "BentoMLContainerClass" implementations out for 0.13.1 tag and 0.13-LTS branch
0.13.1
https://github.com/bentoml/BentoML/blob/a73838ece2d08239df096b60ac6458348c5112fe/bentoml/configuration/containers.py#L209

0.13-LTS
https://github.com/bentoml/BentoML/blob/3954f36762e10f5df15af7e0ae6dd71f5f214261/bentoml/configuration/containers.py#L206


Comparing two "BentoMLContainerClass" implementations above, I think I can't use custom metrics in 0.13.1 the same way I can do in 0.13-LTS, because there isn't static method called metric_client decorated by SingletonFactory in 0.13.1

If there are other way I can use custom metric in 0.13.1, please give me a link to refer
Thank you in advance!

**Comment by yubozhao** [comment]
@heechanlee Thanks for the investigation.  

We patched a few issues on the 0.13-LTS branch and yet to release them.  I think we will release them as `0.13.2` soon.  

For now, if you want to take advantage of the custom metric, you can install BentoML from the source and build bento from there.

```bash
git clone -b 0.13-LTS git@github.com:bentoml/BentoML.git
cd BentoML
pip install -e .
```

**Comment by heechanlee** [comment]
@yubozhao
Thanks a lot!
